### DISCUSSION TITLE
Will the BlueSCSI v2 work with a SD card extension cable?

### DISCUSSION THREAD

**Discussion by dburr** [BlueSCSI/BlueSCSI-v2#55] [Help] [answered]
Subject line says it all. I would like to be able to swap the SD card without having to tear apart my SE/30. On the BlueSCSI v1, there existed a 3D printed mount that would allow you to mount the BlueSCSI in the PDS slot, making the MicroSD card slot accessable from the rear of the machine. However no such mount seems to exist for the BlueSCSI v2. It doesn't look like it would fit anyway. So I was thinking of using a SD card extension cable ([like this one](https://www.amazon.com/fosa-Extender-Extension-Adapter-Flexible/dp/B07DXP3V1R/ref=sr_1_10?crid=Z0CQ1FJBO7X5&keywords=sd+card+extension&qid=1690227307&sprefix=sd+card+extensio%2Caps%2C219&sr=8-10)) which I could either create a PDS mount for, or just leave hanging outside of the case. Has anyone tried something like this? Will it work?

**Comment by erichelgeson** [comment, marked as answer]
There's nothing preventing it from working - though those are usually unsheilded and of wildly varying quality. They are cheap so no reason not to give it a try - I use one in my color classic.

**Reply by ttpilot** [reply]
I have one in my Color Classic as well. Works fine

**Comment by c00gr** [comment]
Mine works fine!
